pcdsdevices/areadetector/detectors.py

```diff
@@ -36,15 +35,13 @@ class PCDSDetector(PCDSDetectorBase):
     IMAGE2: reduced rate image
     Stats2: reduced rate stats
     """
-    image = Cpt(ImagePlugin, ':IMAGE1:', read_attrs=['array_data'])
+    image = Cpt(ImagePlugin, ':IMAGE2:', read_attrs=['array_data'])
     stats = Cpt(StatsPlugin, ':Stats2:', read_attrs=['centroid',
                                                      'mean_value',
                                                      'sigma_x',
                                                      'sigma_y'])
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.image.stage_sigs[self.image.enable] = 1
-        self.stats.stage_sigs[self.stats.enable] = 1
```

Comment:
Can you explain why these are redundant?

Reply:
https://github.com/NSLS-II/ophyd/blob/2cbfccdc54d749e17e87cda0d5d47c129749c2ee/ophyd/areadetector/plugins.py#L70-L71